src/tests/eth/utils.rs

```diff
@@ -0,0 +1,150 @@
+use evm::backend::MemoryAccount;
+use primitive_types::{H160, H256, U256};
+use sha3::{Digest, Keccak256};
+use std::collections::BTreeMap;
+
+pub fn u256_to_h256(u: U256) -> H256 {
```

Comment:
I don't like that we are verbatim copying so [many lines of code](https://github.com/rust-blockchain/evm-tests/blob/master/jsontests/src/utils.rs#L6-L134) from [`evm-jsontests`](https://github.com/rust-blockchain/evm-tests/tree/master/jsontests). Could we add that crate as a dependency instead maybe?

Reply:
We cant' These crate has bugs. For example, when I tried to run tests from that git repo I received errors.

Reply:
Compatibility errors or?

Reply:
Code just can't compile. I mean upstream. So as cate dependency I have the same problem.